certificates.go

```diff
@@ -0,0 +1,119 @@
+package godo
+
+import (
+	"fmt"
+)
+
+const certificatesBasePath = "/v2/certificates"
+
+// CertificatesService is an interface for managing certificates with the DigitalOcean API.
+// See: https://developers.digitalocean.com/documentation/v2/#certificates
+type CertificatesService interface {
+	Get(cID string) (*Certificate, *Response, error)
+	List(opt *ListOptions) ([]Certificate, *Response, error)
+	Create(cr *CertificateRequest) (*Certificate, *Response, error)
+	Delete(cID string) (*Response, error)
+}
+
+// Certificate represents a DigitalOcean certificate configuration.
+type Certificate struct {
+	ID              string `json:"id,omitempty"`
+	Name            string `json:"name,omitempty"`
+	NotAfter        string `json:"not_after,omitempty"`
+	SHA1Fingerprint string `json:"sha1_fingerprint,omitempty"`
+	Created         string `json:"created_at,omitempty"`
+}
+
+// CertificateRequest represents configuration for a new certificate.
+type CertificateRequest struct {
+	Name             string `json:"name,omitempty"`
+	PrivateKey       string `json:"private_key,omitempty"`
+	LeafCertificate  string `json:"leaf_certificate,omitempty"`
+	CertificateChain string `json:"certificate_chain,omitempty"`
+}
+
+type certificateRoot struct {
+	Certificate *Certificate `json:"certificate"`
+}
+
+type certificatesRoot struct {
+	Certificates []Certificate `json:"certificates"`
+	Links        *Links        `json:"links"`
+}
+
+// CertificatesServiceOp handles communication with certificates methods of the DigitalOcean API.
+type CertificatesServiceOp struct {
+	client *Client
+}
+
+var _ CertificatesService = &CertificatesServiceOp{}
+
+// Get an existing certificate by its identifier.
+func (c *CertificatesServiceOp) Get(cID string) (*Certificate, *Response, error) {
+	path := fmt.Sprintf("%s/%s", certificatesBasePath, cID)
```

Comment:
You could use `path.Join` here as well, from the stdlib (though you'd have to rename the `path` var so it doesn't clash with the `path` package).

And for a couple of instances in later methods, too.

Not strictly necessary, but I feel like it's a little more idiomatic and also saves you from accidentally having double slashes.